src/NuGet.Core/NuGet.CommandLine.XPlat/PackCommand.cs

```diff
@@ -0,0 +1,145 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using Microsoft.Dnx.Runtime.Common.CommandLine;
+using NuGet.Commands;
+using NuGet.Logging;
+
+namespace NuGet.CommandLine.XPlat
+{
+    class PackCommand : Command
+    {
+        public PackCommand(CommandLineApplication app, Func<ILogger> getLogger)
+        {
+            app.Command("pack", pack =>
+            {
+                pack.Description = Strings.PackCommand_Description;
+
+                var basePath = pack.Option(
+                    "-b|--basePath <basePath>",
+                    Strings.BasePath_Description,
+                    CommandOptionType.SingleValue);
+
+                var build = pack.Option(
+                    "--build",
+                    Strings.Build_Description,
+                    CommandOptionType.NoValue);
+
+                var excludeEmpty = pack.Option(
+                    "-e|--excludeEmptyDirectories",
+                    Strings.ExcludeEmptyDirectories_Description,
+                    CommandOptionType.NoValue);
+
+                var includeReferencedProjects = pack.Option(
+                    "-e|--includeReferencedProjects",
+                    Strings.IncludeReferencedProjects_Description,
+                    CommandOptionType.NoValue);
+
+                var minClientVersion = pack.Option(
+                    "--minClientVersion <version>",
+                    Strings.MinClientVersion_Description,
+                    CommandOptionType.SingleValue);
+
+                var msBuildVersion = pack.Option(
+                    "--msBuildVersion <version>",
+                    Strings.MsBuildVersion_Description,
+                    CommandOptionType.SingleValue);
+
+                var noDefaultExcludes = pack.Option(
+                    "--noDefaultExcludes",
+                    Strings.NoDefaultExcludes_Description,
+                    CommandOptionType.NoValue);
+
+                var noPackageAnalysis = pack.Option(
+                    "--noPackageAnalysis",
+                    Strings.NoPackageAnalysis_Description,
+                    CommandOptionType.NoValue);
+
+                var outputDirectory = pack.Option(
+                    "-o|--outputDirectory <outputDirectory>",
+                    Strings.OutputDirectory_Description,
+                    CommandOptionType.SingleValue);
+
+                var suffix = pack.Option(
+                    "--suffix <suffix>",
+                    Strings.Suffix_Description,
+                    CommandOptionType.SingleValue);
+
+                var symbols = pack.Option(
+                    "-s|--symbols",
+                    Strings.Symbols_Description,
+                    CommandOptionType.NoValue);
+
+                var tool = pack.Option(
+                    "--tool",
```

Comment:
on xplat a tool package is something else, we should consider dropping some of these options in xplat if possible.